.circleci/config.yml

```diff
@@ -61,31 +54,75 @@ jobs:
             # we will always have drift
             # terraform plan -detailed-exitcode
 
-      - run:
+      - run: &terraform_destroy
           name: Terraform destroy
           working_directory: tests
           command: |
-            terraform destroy -auto-approve
+            terraform destroy --auto-approve || terraform destroy --auto-approve --refresh=false
           when: always
 
-      - run:
+      - run: &terraform_ws_delete
```

Comment:
Has nothing to do with your changes, but if this doesn't run `always` so won't be collecting PR workspaces over time if the workflow errors? Is that ok?

Reply:
I dont think it matters for this one, since its using local state, but Ill add the always for consistency.